Makefile

```diff
@@ -0,0 +1,11 @@
+cloud_e2e_test:
+	bats ./tests/cloud-consumes-data-from-nats.bats
+
+test_fluvio_install:
```

Comment:
What is this for?

Reply:
It's in every connector like that - https://github.com/infinyon/http-source-connector/blob/main/Makefile#L19

Should this be removed ?

Reply:
Ideally, yes, because it makes no sense to keep it here. 

I will remove it from http-source when I have a chance.